### DISCUSSION TITLE
Laravel 8 / PHP 7.4.0 support

### DISCUSSION THREAD

**Discussion by lataupe78** [oddvalue/laravel-drafts#25] [Q&A] [answered]
Hi sir,
 
Is there a way to support Laravel 8 and PHP 7.4.0 ?

Best regards

**Comment by oddvalue** [comment, marked as answer]
Hey @lataupe78 

Unfortunately this package is not able to maintain support for these older versions. There is, however, a fork by another user that drops dependencies down to Laravel 8 and PHP 7.4:

https://github.com/leandrodiogenes/laravel-drafts

If you wish to install that fork you could add this to your composer.json file:

```json

    "repositories": [
        {
            "type": "git",
            "url": "https://github.com/leandrodiogenes/laravel-drafts"
        }
    ]
```

However, I won't be able to provide support if you are not using the official package and that fork is behind this package so not all features will be available.
